skimage/draw/draw3d.py

```diff
@@ -1,20 +1,21 @@
 import numpy as np
-from scipy.special import ellipkinc as ellip_F, ellipeinc as ellip_E
+from scipy.special import elliprg
 
 
 def ellipsoid(a, b, c, spacing=(1.0, 1.0, 1.0), levelset=False):
-    """
-    Generates ellipsoid with semimajor axes aligned with grid dimensions
+    """Generate ellipsoid for given lengths of semi-axes.
+
+    The respective semi-axes are aligned with grid dimensions
```

Comment:
What does aligned with grid dimensions mean? And how do you rotate this thing?

Reply:
Legacy wording, hehe. Looking at the code lines 51-55, it's just a convention: They could/should be called first, second, and third dimensions. Basically it's just that you want the `a, b, c` order to match the `spacing` order.